mqtt-connect/websocket/src/main/java/com/cumulocity/mqtt/connect/client/MqttPublisher.java

```diff
@@ -0,0 +1,21 @@
+package com.cumulocity.mqtt.connect.client;
+
+import com.cumulocity.mqtt.connect.client.model.MqttMessage;
+
+/**
+ * Interface for publishing messages to the server.
+ * <p>
+ * {@link MqttPublisher} instances are setup using {@link MqttClient#buildPublisher(MqttConfig)}.
```

Comment:
suggestion: would the comment be better saying '... instances are created using ...' rather than '.. instances are setup using ...'? Happy to be disagreed with here as it is minor.

Reply:
Completely agree, changed.